backend/src/main/java/de/daniellohrey/betterist/model/DbProduct.java

```diff
@@ -0,0 +1,20 @@
+package de.daniellohrey.betterist.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Document(collection = "userProducts")
+
+public class DbProduct {
+
+    private String _id;
+    private String product_name;
```

Comment:
Mongodb will automatically map `_id` to `id` for you. No snake case for variables please.